src/System.Private.Uri/src/System.Private.Uri.builds

```diff
@@ -9,17 +9,18 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Private.Uri.csproj">
-      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Does the order of Unix vs Windows_NT have any impact? Or is this just an artifact of multiple edits to this file?